e2etest/newe2e_task_runazcopy.go

```diff
@@ -52,13 +53,14 @@ var _ AzCopyStdout = &AzCopyRawStdout{}
 type AzCopyVerb string
 
 const ( // initially supporting a limited set of verbs
-	AzCopyVerbCopy     AzCopyVerb = "copy"
-	AzCopyVerbSync     AzCopyVerb = "sync"
-	AzCopyVerbRemove   AzCopyVerb = "remove"
-	AzCopyVerbList     AzCopyVerb = "list"
-	AzCopyVerbLogin    AzCopyVerb = "login"
-	AzCopyVerbLogout   AzCopyVerb = "logout"
-	AzCopyVerbJobsList AzCopyVerb = "jobs"
+	AzCopyVerbCopy        AzCopyVerb = "copy"
+	AzCopyVerbSync        AzCopyVerb = "sync"
+	AzCopyVerbRemove      AzCopyVerb = "remove"
+	AzCopyVerbList        AzCopyVerb = "list"
+	AzCopyVerbLogin       AzCopyVerb = "login"
+	AzCopyVerbLogout      AzCopyVerb = "logout"
+	AzCopyVerbJobsList    AzCopyVerb = "jobs"
+	AzCopyVerbJobsCleanup AzCopyVerb = "jobs"
```

Comment:
Instead of having multiple that represent the same string, should we just make one called AzCopyVerbJobs?

Reply:
Addressed